mkdocs.yml

```diff
@@ -86,7 +86,29 @@ nav:
         - iBeacon&trade; Driver: connect-field-devices/ibeacon-driver.md
         - Eddystone&trade; Driver: connect-field-devices/eddystone-driver.md
         - Xdk Driver: connect-field-devices/xdk-driver.md
-
+    - Kura Wires:
+      - Introduction: kura-wires/introduction.md
+      - Assets as Wire Components: kura-wires/assets-as-wire-components.md
+      - DB Store and Filter: kura-wires/db-store-and-filter.md
+      - FIFO Component: kura-wires/fifo-component.md
+      - Multiport Wire Components:
+        - Multiport Wire Components: kura-wires/multiport-wire-components/multiport-wire-components.md
+        - Conditional Component: kura-wires/multiport-wire-components/conditional-component.md
+        - Join Component: kura-wires/multiport-wire-components/join-component.md
+        - Mathematical Components Example: kura-wires/multiport-wire-components/mathematical-components-example.md
+      - AI Wire Component: kura-wires/ai-wire-component.md
+      - Script Filter: kura-wires/script-filter.md
+      - Usage Examples:
+        - Modbus Application: kura-wires/usage-examples/modbus-application.md
+        - OPC-UA Application: kura-wires/usage-examples/opcua-application.md
+        - TI SensorTag Application: kura-wires/usage-examples/ti-sensortag-application.md
+        - GPIO Driver Application: kura-wires/usage-examples/gpio-driver-application.md
+        - iBeacon&trade; Driver Application: kura-wires/usage-examples/ibeacon-driver-application.md
+        - Eddystone&trade; Driver Application with RuuviTag+: kura-wires/usage-examples/eddystone-driver-application.md
+      - Wires MQTT Namespace: kura-wires/wires-mqtt-namespace.md
+      - WireGraphService Configuration Format: kura-wires/wire-graph-service-configuration-format.md
```

Comment:
Grouping the Wire Components together in a sub-section can help navigation? Something like this:

```suggestion
      - Introduction: kura-wires/introduction.md
      - Assets as Wire Components: kura-wires/assets-as-wire-components.md
      - Multiport Wire Components:
        - Multiport Wire Components: kura-wires/multiport-wire-components/multiport-wire-components.md
        - Conditional Component: kura-wires/multiport-wire-components/conditional-component.md
        - Join Component: kura-wires/multiport-wire-components/join-component.md
        - Mathematical Components Example: kura-wires/multiport-wire-components/mathematical-components-example.md
      - Special Wire Components:
        - AI Wire Component: kura-wires/ai-wire-component.md
        - Script Filter: kura-wires/script-filter.md
        - DB Store and Filter: kura-wires/db-store-and-filter.md
        - FIFO Component: kura-wires/fifo-component.md
      - Usage Examples:
        - Modbus Application: kura-wires/usage-examples/modbus-application.md
        - OPC-UA Application: kura-wires/usage-examples/opcua-application.md
        - TI SensorTag Application: kura-wires/usage-examples/ti-sensortag-application.md
        - GPIO Driver Application: kura-wires/usage-examples/gpio-driver-application.md
        - iBeacon&trade; Driver Application: kura-wires/usage-examples/ibeacon-driver-application.md
        - Eddystone&trade; Driver Application with RuuviTag+: kura-wires/usage-examples/eddystone-driver-application.md
      - Wires MQTT Namespace: kura-wires/wires-mqtt-namespace.md
      - WireGraphService Configuration Format: kura-wires/wire-graph-service-configuration-format.md
```